logging/README.md

```diff
@@ -179,19 +193,17 @@ make in the `user-values-fluent-bit-open.yaml` file that are intended to
 affect the Fluent Bit configuration files are ignored. However, edits
 affecting other aspects of the Fluent Bit Helm chart execution are processed.
 
-When you edit the `user-values-fluent-bit.yaml` file, ensure that the parent
+When you edit the `user-values-opensearch.yaml` file, ensure that the parent
 item of any item that you uncomment is also uncommented.  For
-example, if you uncommented the `storageClass` item for the Elasticsearch
+example, if you uncommented the `storageClass` item for the OpenSearch
 master nodes, you must also uncomment the `persistence` item, the
 `master` item and the `elasticsearch` item, as shown below:
 
 ```yaml
-# Sample user-values-elasticsearch-open.yaml
+# Sample user-values-opensearch-open.yaml
```

Comment:
Change to: # Sample user-values-opensearch.yaml

Reply:
Correction made.